operations/customers.md

```diff
@@ -142,7 +142,7 @@ Returns all customers from the specified interval according to the time filter \
 * `VeryImportant`
 * `Problematic`
 * `Cashlist`
-* `Disabled`
+* `Disabled person`
```

Comment:
This is incorrect. You will not get the value with space from the API.
Correct fix is: `DisabledPerson`

Reply:
Thanks!